src/test/node/node-runtime.test.ts

```diff
@@ -110,6 +110,33 @@ describe('node runtime', () => {
     handle.assertLog({ substring: true });
   });
 
+  itIntegrates('chakracore string value', async ({ r }) => {
+    createFileTree(testFixturesDir, {
```

Comment:
You'll want to bail out here if `process.platform !== 'win32'`

Reply:
Thanks.